tests/unit_test/apis/impl/controller_test.py

```diff
@@ -1148,7 +1148,7 @@ def test_task_only_exit_when_min_responses_received(self, method, min_responses)
 
     @pytest.mark.parametrize("min_responses", [1, 2, 3, 4])
     @pytest.mark.parametrize("wait_time_after_min_received", [1, 2])
-    def test_task_only_exit_when_wait_time_after_min_received(
+    def test_task_exit_quickly_when_all_responses_received(
         self, method, min_responses, wait_time_after_min_received
     ):
```

Comment:
Do we want to just get rid of "wait_time_after_min_received"?

Reply:
Yes, because it is invalid. We could add another test case to really test "wait min time after some (but not all) are received".